pkg/cmd/eventlistener/list.go

```diff
@@ -107,18 +107,28 @@ func list(client versioned.Interface, namespace string) (*v1alpha1.EventListener
 	return els, nil
 }
 
+func formatAddress(el v1alpha1.EventListener) string {
```

Comment:
Can we move this to format pkg and use https://github.com/tektoncd/cli/blob/master/pkg/cmd/eventlistener/describe.go#L226 for URL